src/App.js

```diff
@@ -1,9 +1,13 @@
 import './style.css';
 import Calculator from './components/Calculator';
+import FetchAPIData from './components/Fetch_API_Data';
 
 function App() {
   return (
-    <Calculator />
+    <div className="wrapper">
+      <Calculator />
+      <FetchAPIData />
+    </div>
```

Comment:
- [ ] Then make sure the `<Quotes>` component is imported into the main component. Example:

```jsx
import Calculator from "./components/Calculator"
import Quotes from "./components/Quotes"

function App() {
  return (
    <>
      {/* Rest of JSX */}
      <Calculator />
      <Quotes />
    </>
  )
}

export default App
```